keras_retinanet/bin/train.py

```diff
@@ -202,7 +208,8 @@ def parse_args(args):
     csv_parser.add_argument('classes', help='Path to a CSV file containing class label mapping.')
     csv_parser.add_argument('--val-annotations', help='Path to CSV file containing annotations for validation (optional).')
 
-    parser.add_argument('--weights',       help='Weights to use for initialization (defaults to ImageNet).', default='imagenet')
+    parser.add_argument('--weights',       help='Weights to use for initialization (defaults to \'imagenet\').', default='imagenet')
+    parser.add_argument('--snapshot',      help='Snapshot to resume training with.')
```

Comment:
We could use a mutually exclusive group for `--weights` and `--snapshot`: https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
Perfect! Added it.